gems/quilt_rails/lib/quilt_rails.rb

```diff
@@ -2,6 +2,7 @@
 module Quilt
 end
 
+require "quilt_rails/engine"
 require "quilt_rails/version"
 require "quilt_rails/configuration"
 require "quilt_rails/railtie"
```

Comment:
You can move everything that is inside the Railtie to the Engine and delete the Railtie.